packages/amplify-appsync-simulator/src/velocity/util/time.ts

```diff
@@ -1,32 +1,89 @@
-import * as dateformat from 'dateformat';
+import * as moment from 'moment';
+import 'moment-timezone';
+import 'moment-jdateformatparser';
 
-export const time = now => ({
-  nowISO8601() {
-    return now.toISOString();
+declare module 'moment' {
+  export interface Moment {
+    toMomentFormatString: (format: string) => string;
+    formatWithJDF: (format: string) => string;
+  }
+}
+
+function parseTimeStamp(dateTime: string, format?: string, timezone?: string): moment.Moment {
```

Comment:
typo: "parseTimestamp", lowercase s in stamp.

TS specific:
```
const parseTimestamp = (dateTime: string, format?: string, timezone?: string): moment.Moment => {
...
}
```

instead of ```function```

Reply:
Is there any benefit in converting this to an arrow function instead of regular old function